### ISSUE TITLE
Add CKAN support

### ISSUE THREAD

**Issue by jqnatividad** [timwis/vizwit#170]
No-brainer as not only does OpenDataPhilly run CKAN, WPRDC does as well.

This functionality is also much needed in other CKAN installations.

**Comment by timwis** [comment]
Definitely. @mheadd was talking about working on this too. It would probably be pretty similar to the [CartoDB provider](#171) since it's just SQL. We also have a [wiki page outlining the spec](https://github.com/timwis/vizwit/wiki/Adding-a-provider).

**Comment by mheadd** [comment]
@jqnatividad @rgradeck This would be perfect for WPRDC. Agree with @timwis that a CKAN provider would be very similar to the CartoDB provider. Alas, my bandwidth to work on this has to date been limited. Will reach out to you to coordinate - maybe we can tag team this.

**Comment by jqnatividad** [comment]
Definitely tag-team.  And thanks for the pointer to the CartoDB provider and the provider spec.  This should give us a running start.

**Comment by rgradeck** [comment]
Love it. If there's anything we can do to contribute, we'd be happy to do so.

**Comment by rgradeck** [comment]
Cc'ing @saylorsd
